src/components/SimplePriceChart/SimplePriceChart.tsx

```diff
@@ -104,4 +167,18 @@ const ChartTitle = styled.h2`
   font-size: 42px;
 `
 
+const DurationButton = styled.button`
+  background-color: radial-gradient(
+    circle at center top,
+    rgb(22, 20, 31),
+    rgb(22, 20, 31)
+  );
```

Comment:
this radial gradient doesn't seem to be showing up very clearly? unless I'm missing something...
If not maybe remove?